### PR TITLE
Rugby - Parse qualifiers table and display them for qualifiers game

### PR DESCRIPTION
**Pull request by mchv**
The associated changes add qualifiers table for qualifiers matches [like on football matches](http://www.theguardian.com/football/live/2015/sep/16/barcelona-v-roma-champions-league-live), just ready for the world cup start :rugby_football: !  

screen shot 2015-09-17 at 23 43 00
- Few nice details:
  - Clicking on the group name redirect to the [interactive chart](http://www.theguardian.com/sport/ng-interactive/2015/sep/07/rugby-world-cup-2015-digital-wallchart)
    screen shot 2015-09-17 at 23 43 06
  - We display a divider to indicates the (virtually) qualified teams for the knock-out stage
  
  screen shot 2015-09-17 at 23 43 14
  - display correctly on mobile (at the end of the article) like match stats 

The ui is a switch, to test locally you can override `RugbyStatsJob.getGroupTable(rugbyMatch: Match)` to the following:

``` scala
def getGroupTable(rugbyMatch: Match): Option[GroupTable] = {
    groupTables.get.get(rugby.feed.WorldCup2015).flatMap(_.headOption)
}
```

Finally, because as [gentlemen we never fire first](https://en.wikipedia.org/wiki/Battle_of_Fontenoy), @jennysivapalan I let you [start the hostility](https://github.com/guardian/frontend/pull/10555), but let me reply with this excerpt from [today lemonde.fr article](http://www.lemonde.fr/coupe-du-monde-rugby/article/2015/09/17/dix-bonnes-raisons-de-regarder-la-coupe-du-monde-de-rugby_4761237_1616920.html) explaining why one should follow the world cup:

screen shot 2015-09-18 at 00 13 03

:rooster: :fr: :rugby_football:

### CODE REVIEW

File: sport/app/rugby/jobs/RugbyStatsJob.scala [hunk 1]
```diff
@@ -37,7 +37,7 @@ trait RugbyStatsJob extends ExecutionContexts with Logging {
       })
     }.recover {
       case optaFeedException: RugbyOptaFeedException => log.warn(s"RugbyStatsJob encountered errors: ${optaFeedException.message}")
-      case error: Exception => log.warn(error.getMessage)
+      case error: Exception => log.warn(error.getMessage, error)
```
Comment (jennysivapalan) [inline sport/app/rugby/jobs/RugbyStatsJob.scala]: Do we need to log the `error`?

Comment (mchv) [inline sport/app/rugby/jobs/RugbyStatsJob.scala]: If we log the `error` we have the stacktrace, otherwise it is lost

Comment (jennysivapalan) [inline sport/app/rugby/jobs/RugbyStatsJob.scala]: Ok.

**Comment by jennysivapalan** [comment]
Looks good :+1: I think just as it's quite close to the first game and `getGroupTable` hasn't been tested under the 'real' conditions I'd be tempted to just turn this on in CODE for the weekend and if all is good turn it on for PROD on Monday. Up to you though.

**Pull request merged**